### PR TITLE
SSHDataGrabber outputs now returns full path to the grabbed files

### PR DESCRIPTION
**Pull request by demianw**
Corrected an issue in which the path to the downloaded files was not returned in full. Hence the pipelines were not working

### CODE REVIEW

**Comment by satra** [comment]
@demianw - could you please update the CHANGES file?

**Comment by demianw** [comment]
@satra done!

File: CHANGES [hunk 1]
```diff
@@ -50,6 +50,7 @@ Next release
 * ENH: ANTs JointFusion() (https://github.com/nipy/nipype/pull/1042)
 * ENH: Added csvReader() utility (https://github.com/nipy/nipype/pull/1044)
 * FIX: typo in nipype.interfaces.freesurfer.utils.py Tkregister2 (https://github.com/nipy/nipype/pull/1083)
+* FIX: SSHDataGrabber outputs now return full path to the grabbed/downloaded files.
```
Comment (satra) [inline CHANGES]: add URL : https://github.com/nipy/nipype/pull/1086

we are trying to create a direct link back from the changes so people can see the exact changes.

**Comment by demianw** [comment]
Great idea! Done

**Comment by satra** [comment]
push :)

**Comment by demianw** [comment]
Done ;)

**Pull request merged**